### DISCUSSION TITLE
说实话我看到这个的第一反应是……

### DISCUSSION THREAD

**Discussion by fierce-long** [share121/inter-knot#1038] [其他]
![《绝区零》青衣角色展示 _ 青童与白叟_哔哩哔哩_bilibili 和另外 1 个页面 - 个人 - Microsoft​ Edge 2024_8_10 21_26_17](https://github.com/user-attachments/assets/41a40c0d-747a-4a94-bd8c-46ff6789fd30)
形不成形，意不在意，再去练练吧。
千招百式在一息
劲发江潮落，气收秋毫平
你们解决问题，还是只会仰仗干戈吗？
征鼓一声千军动，掬罢黄沙浣铁衣

**Comment by lingyushen** [comment]
<h1 style="color: #2ecc71">大哥何时来的</h1>
